src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt

```diff
@@ -0,0 +1,18 @@
+package hs.kr.equus.user.global.utils.pass
+
+import hs.kr.equus.user.domain.auth.exception.InvalidUrlException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class RedirectUrlChecker {
+
+    @Value("\${pass.base-url}")
+    private lateinit var BASE_URL: String
```

Comment:
🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case